.eslintignore

```diff
@@ -1,3 +1,4 @@
 !/.eslint.js
 /test/**/*.js
 !/test/index.js
+*.mjs
```

Comment:
This is probably not the best solution. The problem is that I can add an `"extends"` directive inside an `"overrides"` directive in `.eslintrc` files.